keras_nlp/models/deberta/deberta_backbone.py

```diff
@@ -0,0 +1,274 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DeBERTa backbone models."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.models.deberta.deberta_encoder import DebertaEncoder
+from keras_nlp.utils.python_utils import classproperty
+
+
+def deberta_kernel_initializer(stddev=0.02):
+    return keras.initializers.TruncatedNormal(stddev=stddev)
+
+
+class RelativeEmbedding(keras.layers.Layer):
+    """Relative embedding layer.
+
+    This layer initializes an embedding matrix (of shape
+    `(2 * batch_size, hidden_dim)`) for relative position encoding. It then
+    applies layer normalization on the embedding matrix and returns the relative
+    embedding matrix. This is in accordance with the original DeBERTa V3
```

Comment:
Maybe rather than this in accordance sentence, you could reword this to be more similar to the docstring you have for the self attention layer.


This is an implementation of a relative embedding matrix as described in the paper ["DeBERTaV3: Improving DeBERTa using ELECTRA-Style Pre-Training with Gradient-Disentangled Embedding Sharing"](https://arxiv.org/abs/2111.09543). This layer...